admin/class-sitemaps-admin.php

```diff
@@ -98,4 +111,71 @@ function status_transition( $new_status, $old_status, $post ) {
 			}
 		}
 	}
+
+	/**
+	 * While bulk importing, just save unique post_types
+	 *
+	 * When importing is done, if we have a post_type that is saved in the sitemap
+	 * try to ping the search engines
+	 *
+	 * @param string   $new_status New post status.
+	 * @param string   $old_status Old post status.
+	 * @param \WP_Post $post       Post object.
+	 */
+	private function status_transition_bulk( $new_status, $old_status, $post ) {
+		if ( ! defined( 'WP_IMPORTING' ) ) {
```

Comment:
Why are you checking if `WP_IMPORTING` is not defined when this also already happens on line [71](https://github.com/Yoast/wordpress-seo/pull/3888/files#diff-795da391d41ec2f03fb11b360c56b573R71)?